src/help.js

```diff
@@ -48,9 +57,10 @@ folder name used already exists.
 export const notRecognised = () => {
 console.log(
 `
-Flag(s) not recognised. Use the help command below for more info:
+Flag(s) not recognised. Use any of the help command below for more info:
 
-node-mongo --help`
+node-mongo --help
+nmgo --help`
 );
```

Comment:
**Review comment 2 of 2:**

Changes cleanly added 👍🏽 Your file changes shows this.

For the flag command info (`notRecognised` function), you can just use ideas from the changes made to the `help` function. Something like `Help command: ` and `Help command (for shorter command alternative): `

Reply:
Just so it is consistent.

Reply:
I have made the suggested changes and push for review.

Reply:
Thank you